src/gridstack.ts

```diff
@@ -529,6 +529,7 @@ export class GridStack {
         delete o.alwaysShowResizeHandle;
       }
       Utils.removeInternalAndSame(o, GridDefaults);
+      o.float = this.engine.float

```

Comment:
no I would do it on float(true) API to set them both instead. user might never call save.

Reply:
@adumesny changed. I also checked if the value actually changed so that `_triggerChangeEvents` wasn't called when nothing changed, but I can see this maybe being undesired too. Lemme know.